src/Microsoft.SqlTools.ServiceLayer/Security/UserData.cs

```diff
@@ -1044,46 +1044,47 @@ public UserPrototype GetUserPrototype(ExhaustiveUserTypes userType)
+                //     this.currentPrototype = certificateMappedUser; 
+                //     break;
 
                 case ExhaustiveUserTypes.LoginMappedUser:
                     currentData.userType = UserType.SqlUser;
                     this.loginMappedUser ??= new UserPrototypeForSqlUserWithLogin(this.context, this.currentData, this.originalData);
                     this.currentPrototype = loginMappedUser;
                     break;
 
-                case ExhaustiveUserTypes.SqlUserWithoutLogin:
-                    currentData.userType = UserType.NoLogin;
-                    this.noLoginUser ??= new UserPrototypeWithDefaultSchema(this.context, this.currentData, this.originalData);
-                    this.currentPrototype = noLoginUser;
-                    break;
-
-                case ExhaustiveUserTypes.SqlUserWithPassword:
-                    currentData.userType = UserType.SqlUser;
-                    this.sqlUserWithPassword ??= new UserPrototypeForSqlUserWithPassword(this.context, this.currentData, this.originalData);
-                    this.currentPrototype = sqlUserWithPassword;
-                    break;
-
-                case ExhaustiveUserTypes.WindowsUser:
-                    currentData.userType = UserType.SqlUser;
-                    this.windowsUser ??= new UserPrototypeForWindowsUser(this.context, this.currentData, this.originalData);
-                    this.currentPrototype = windowsUser;
-                    break;
+                // case ExhaustiveUserTypes.SqlUserWithoutLogin:
+                //     currentData.userType = UserType.NoLogin;
+                //     this.noLoginUser ??= new UserPrototypeWithDefaultSchema(this.context, this.currentData, this.originalData);
+                //     this.currentPrototype = noLoginUser;
+                //     break;
+
+                // case ExhaustiveUserTypes.SqlUserWithPassword:
+                //     currentData.userType = UserType.SqlUser;
+                //     this.sqlUserWithPassword ??= new UserPrototypeForSqlUserWithPassword(this.context, this.currentData, this.originalData);
+                //     this.currentPrototype = sqlUserWithPassword;
+                //     break;
+
+                // case ExhaustiveUserTypes.WindowsUser:
+                //     currentData.userType = UserType.SqlUser;
+                //     this.windowsUser ??= new UserPrototypeForWindowsUser(this.context, this.currentData, this.originalData);
+                //     this.currentPrototype = windowsUser;
+                //     break;
                 
                 default:
                     System.Diagnostics.Debug.Assert(false, "Unknown UserType provided.");
                     this.currentPrototype = null;
-                    break;
+                    throw new ApplicationException("Only 'User with Login' user type support in March Preview release");
```

Comment:
probably we don't have to call out the release?

Reply:
I'm assuming this won't be the case in the next release :)